docs/managing.md

```diff
@@ -78,6 +83,76 @@ configurable settings.
 
 See the [document on upgrading](upgrading.md).
 
+## Failure handling
+
+When using local storage, a Cassandra pod is using a local persistent volume
+that is only available when the pod is scheduled in a specific node. Any
+rescheduling will land the pod to the very same node due to the volume node
+affinity.
+
+This is an issue in case of a total Kubernetes node loss: the pods running on an
+unreachable Node enter the states Terminating or Unknown. Kubernetes doesn’t
+allow the deletion of those pods to avoid any brain-split.
+
+KUDO Cassandra provides a way to automatically handle these failure modes and
+move a Cassandra node that is located on a failed Kubernetes node to a different
+node in the cluster.
+
+### Recovery controller
+
+To enable this feature, use the following parameter:
+
+```bash
+RECOVERY_CONTROLLER=true
+```
+
+When this parameter is set, KUDO Cassandra will deploy an additional controller
+that monitors the deployed Cassandra pods. If any pod reaches an unschedulable
+state and detects that the kubernetes node is gone, it will remove the local
+volume of that pod and allow Kubernetes to schedule the pod to a different node.
+Additionally, the rescheduling can be triggered by an eviction label.
+
+The recovery controller relies on the Kubernetes state of a node, not the actual
+running processes. This means that the failure of the hardware on which a
+Cassandra node runs does not trigger the recovery. The only way an automatic
+recovery is triggered is when the Kubernetes node is removed from the cluster by
+kubectl delete node <failed-node-name>. This allows a Kubernetes node to be shut
+down for a maintenance period without KUDO Cassandra triggering a recovery.
+
+:warning: This feature will remove persistent volume claims in the Kubernetes
+cluster. This may lead to data loss. Additionally, you must not use any
+keyspaces with a replication factor of ONE, or the data of the failed Cassandra
+node will be lost.
+
+#### Node eviction
+
+Evicting a Cassandra node is similar to Failure recovery described above. The
+recovery controller will automate certain steps. The main difference is that
+during node eviction the Kubernetes node should stay available, i.e. other pods
+on that node shouldn’t get evicted. To evict a Cassandra node, first cordon or
+taint the Kubernetes node the Cassandra node is running on. This ensures that
```

Comment:
We can now also add a section to describe that KUDO Cassandra supports the cordon for Cassandra only.

If you want to cordon only Cassandra from a specific node, add a label `kudo-cassandra/cordon=true` to that node. KUDO Cassandra pods will not be scheduled to Kubernetes nodes with this label present.

Reply:
`s/the cordon for Cassandra only/the cordon specific to KUDO-Cassandra/`